test/CMakeLists.txt

```diff
@@ -81,68 +81,72 @@ add_test(NAME ArborX_Test_Geometry COMMAND ArborX_Test_Geometry.exe)
 
 set(ARBORX_TEST_QUERY_TREE_SOURCES)
 foreach(_test Callbacks Degenerate ManufacturedSolution ComparisonWithBoost)
-  file(WRITE "${CMAKE_CURRENT_BINARY_DIR}/tstQueryTree${_test}_BVH.cpp.tmp"
-    "#include <ArborX_LinearBVH.hpp>\n"
-    "#include <ArborX_Box.hpp>\n"
-    "#include \"ArborXTest_LegacyTree.hpp\"\n"
-    "template <class MemorySpace>\n"
-    "using ArborX_Legacy_BVH_Box =\n"
-    "    LegacyTree<ArborX::BoundingVolumeHierarchy<\n"
-    "        MemorySpace, ArborX::PairValueIndex<ArborX::Box<3, float>>,\n"
-    "        ArborX::Experimental::DefaultIndexableGetter, ArborX::Box<3, float>>>;\n"
-    "#define ARBORX_TEST_TREE_TYPES Tuple<ArborX_Legacy_BVH_Box>\n"
-    "#define ARBORX_TEST_DEVICE_TYPES std::tuple<${ARBORX_DEVICE_TYPES}>\n"
-    "#include <tstQueryTree${_test}.cpp>\n"
-  )
-  configure_file(
-    "${CMAKE_CURRENT_BINARY_DIR}/tstQueryTree${_test}_BVH.cpp.tmp"
-    "${CMAKE_CURRENT_BINARY_DIR}/tstQueryTree${_test}_BVH.cpp" COPYONLY
-  )
-  list(APPEND ARBORX_TEST_QUERY_TREE_SOURCES "${CMAKE_CURRENT_BINARY_DIR}/tstQueryTree${_test}_BVH.cpp")
-  file(WRITE "${CMAKE_CURRENT_BINARY_DIR}/tstQueryTree${_test}_BF.cpp.tmp"
-    "#include <ArborX_BruteForce.hpp>\n"
-    "#include <ArborX_Box.hpp>\n"
-    "#include \"ArborXTest_LegacyTree.hpp\"\n"
-    "template <class MemorySpace>\n"
-    "using ArborX_Legacy_BruteForce_Box =\n"
-    "    LegacyTree<ArborX::BruteForce<\n"
-    "        MemorySpace, ArborX::PairValueIndex<ArborX::Box<3, float>>,\n"
-    "        ArborX::Experimental::DefaultIndexableGetter, ArborX::Box<3, float>>>;\n"
-    "#define ARBORX_TEST_TREE_TYPES Tuple<ArborX_Legacy_BruteForce_Box>\n"
-    "#define ARBORX_TEST_DEVICE_TYPES std::tuple<${ARBORX_DEVICE_TYPES}>\n"
-    "#define ARBORX_TEST_DISABLE_CALLBACK_EARLY_EXIT\n"
-    "#include <tstQueryTree${_test}.cpp>\n"
-  )
-  configure_file(
-    "${CMAKE_CURRENT_BINARY_DIR}/tstQueryTree${_test}_BF.cpp.tmp"
-    "${CMAKE_CURRENT_BINARY_DIR}/tstQueryTree${_test}_BF.cpp" COPYONLY
-  )
-  list(APPEND ARBORX_TEST_QUERY_TREE_SOURCES "${CMAKE_CURRENT_BINARY_DIR}/tstQueryTree${_test}_BF.cpp")
-  foreach(_bounding_volume KDOP14 KDOP18) # purposefully ommitting KDOP6 and KDOP26 to reduce the number of instantiations
-    file(WRITE "${CMAKE_CURRENT_BINARY_DIR}/tstQueryTree${_test}_BVH_${_bounding_volume}.cpp.tmp"
+  foreach (_precision float double)
```

Comment:
By how much are we increasing compile time? runtime?
I am interested both in these numbers for the QueryTree exe and total times